src-lib/darknet.h

```diff
@@ -742,7 +742,12 @@ typedef struct network {
 	int outputs;
 	int truths;
 	int notruth;
-	int h, w, c;
+	// The height of the network
```

Comment:
I appreciate breaking out the variables one per line, and adding comments.  When you do this to one of the .h or .hpp files, please use Doxygen-style comments so it can be picked up and shown in the Doxygen output.